Documentation/network/concepts/ipam/azure.rst

```diff
@@ -203,6 +203,11 @@ When a node or instance terminates, the Kubernetes apiserver will send a node
 deletion event. This event will be picked up by the operator and the operator
 will delete the corresponding ``ciliumnodes.cilium.io`` custom resource.
 
+Masquerading
+================
+
+Masquerading is supported via the eBPF :ref:`ip-masq-agent <concepts_masquerading>` or by setting ``--ipv4-native-routing-cidr``
```

Comment:
Thanks! I just looked at the doc change, could you please quickly add the missing punctuation?

```suggestion
Masquerading is supported via the eBPF :ref:`ip-masq-agent <concepts_masquerading>` or by setting ``--ipv4-native-routing-cidr``.
```